dynamo/configuration.py

```diff
@@ -3,17 +3,100 @@
 from matplotlib import rcParams, cm, colors
 from cycler import cycler
 import matplotlib.pyplot as plt
+from .dynamo_logger import main_info
+
+
+class DynamoConfig:
```

Comment:
name the class to DynamoSaveConfig

There are a lot of other configuration here